source/administration/bulk-export-data.rst

```diff
@@ -0,0 +1,322 @@
+Bulk export data
+=================
+At this time, core properties of non-deleted objects listed below are exported.  Configuration for exporting specific areas of the server, exporting additional types of posts, reactions, file attachments, webhooks and bot messages are not yet supported. 
+
+Version object
```

Comment:
@grundleborg Can you confirm version is included in the export? I believe this is required by the import functionality, but it may just need to be cut from this document if it is not exported explicitly from this command?